Doc/library/curses.ascii.rst

```diff
@@ -176,7 +176,7 @@ C library:
 
    Checks for a non-ASCII character (ordinal values 0x80 and above).
 
-These functions accept either integers or strings; when the argument is a
+These functions accept either integers or single-character strings; when the argument is a
```

Comment:
Does `single-character string` as same as you mention in note? (`it can be specified as a Unicode string or a byte string.`). If yes, but these function can't accept byte string:

```python
>>> curses.ascii.ascii(b'1')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.6/curses/ascii.py", line 75, in ascii
    return _ctoi(c) & 0x7f
TypeError: unsupported operand type(s) for &: 'bytes' and 'int'
>>> curses.ascii.ctrl(b'1')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.6/curses/ascii.py", line 81, in ctrl
    return _ctoi(c) & 0x1f
TypeError: unsupported operand type(s) for &: 'bytes' and 'int'
```